src/app/components/import/ImportModal.js

```diff
@@ -0,0 +1,131 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { c } from 'ttag';
+import { useNotifications, FormModal, useUser, useUserKeys } from 'react-components';
+
+import ImportFooter from './ImportFooter';
+import AttachingModalContent from './AttachingModalContent';
+import ImportCsvModalContent from './ImportCsvModalContent';
+import ImportingModalContent from './ImportingModalContent';
+import ImportGroupsModalContent from './ImportGroupsModalContent';
+
+import { IMPORT_STEPS } from '../../constants';
```

Comment:
If you do https://github.com/ProtonMail/proton-contacts/pull/15/files#r304797105 you can move the `IMPORT_STEPS` back from constants to here because I think it does not make sense to define them in `constants` since it's not relevant for the whole application